### PR TITLE
fixed shared range

### PR DESCRIPTION
**Pull request by jvinolas**
Fixed range with shared network that was out of network definition in shared

### CODE REVIEW

**Review by sim6 — COMMENTED** [review]
Why we leave 799 hosts outside the DHCP?

**Comment by jvinolas** [comment]
Which ones?

    <ip address='192.168.124.1' netmask='255.255.252.0'>
    <dhcp>
      <range start='192.168.127.33' end='192.168.127.254'/>
    </dhcp>
  </ip>

File: docker/hypervisor/networks/shared.xml [hunk 1]
```diff
@@ -4,9 +4,9 @@
   <forward mode='nat'/>
   <bridge name='virbr1' stp='on' delay='0'/>
   <mac address='52:54:00:47:b3:70'/>
-  <ip address='192.168.123.1' netmask='255.255.255.0'>
+  <ip address='192.168.124.1' netmask='255.255.252.0'>
     <dhcp>
-      <range start='192.168.123.2' end='192.168.123.254'/>
+      <range start='192.168.127.33' end='192.168.127.254'/>
```
Comment (sim6) [inline docker/hypervisor/networks/shared.xml:9]: From 192.168.124.1 to 192.168.127.33 are 799 addresses. To leave a 32 hosts outside of  DHCP, we would start the DHCP range to 192.168.124.33.

```suggestion
      <range start='192.168.124.33' end='192.168.127.254'/>
```

**Review by sim6 — CHANGES_REQUESTED** [review]

**Comment by sim6** [comment]
We are leaving 799 hosts outside DHCP range and only 222 host for the DHCP range. :(

**Pull request merged**